chocolatey/copenhagen_theme
# (#20) Enable TypeScript

## Description Of Changes
The gulpfile.js file has been updated to allow the compilation of TypeScript when running `gulp`. All other changes to allow this conversion, including the tsconfig.json file, are located in choco-theme.

## Motivation and Context
These changes will allow the usage and compilation of TypeScript.

## Testing
1. Run `yarn install --immutable`
2. Run `gulp`
3. The gulp task should complete. Note that there will be scss warnings in the console. This is expected and not related.

### Operating Systems Testing
n/a

## Change Types Made

## Change Checklist


## Related Issue
* PR: https://github.com/chocolatey/choco-theme/pull/326
* ENGTASKS-3118
* #20
